libs/features/mas/web-components/test/merch-card.ccd-suggested.test.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Merch Card CCD Suggested Web Component demo page</title>
+    <script>
+        window.process = {
+            env: {},
+        };
+    </script>
+    <style>
+        main {
+            display: flex;
+            align-items: center;
+            gap: 20px;
+            flex-direction: column;
+            padding-top: 80px;
+            width: 100%;
+            height: 100%;
+        }
+        merch-card {
+          margin-bottom: 20px;
+        }
+    </style>
+</head>
+
+<body>
+    <script type="module" src="./merch-card.ccd-suggested.test.html.js"></script>
+    <main>
+      <sp-theme theme="spectrum" color="light" scale="medium" dir="ltr">
+          <merch-card class="" tabindex="0" variant="ccd-suggested" filters="" types="">
+            <sp-status-light size="l" variant="info" dir="ltr"></sp-status-light>
+            <merch-icon
+              slot="icons"
+              src="https://www.adobe.com/content/dam/shared/images/product-icons/svg/photoshop.svg"
```

Comment:
test files should not reference external resources, they will be logged as warnings. 
I recommend to use the pixel image.